packages/bpk-docs/src/constants/routes.js

```diff
@@ -100,6 +100,7 @@ export const TOUCHABLE_NATIVE_FEEDBACK =
   '/components/touchable-native-feedback';
 export const MAP = '/components/map';
 export const CAROUSEL = '/components/carousel';
+export const INFINITE_SCROLL = '/components/infiniteScroll';
```

Comment:
This should be in kebab case to match convention, e.g. we have `/components/touchable-native-feedback`